### PR TITLE
Add my module.  - Consultado

### PR DESCRIPTION
**Pull request by consultado-kirby**
It contains:
1. Personal information
2. My Favorite Quotes
3. Motivational Story

### CODE REVIEW

File: ourpackage/consultado.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import os
+
+def kirby():
+    while True:
+        os.system("cls")
+        print(Fore.GREEN + "    SOMETHING ABOUT ME")
+        print(Fore.RESET + "\n 1. Personal information")
+        print(" 2. My Favorite Quotes")
+        print(" 3. Motivational Story")
+        print(Fore.RED + "\n                    0 - Exit")
```
Comment (barcelos-kevin) [inline ourpackage/consultado.py:15]: I will add my changes/comment. - Barcelos

**Review by barcelos-kevin — CHANGES_REQUESTED** [review]

File: ourpackage/consultado.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import os
+
+def kirby():
+    while True:
+        os.system("cls")
+        print(Fore.GREEN + "    SOMETHING ABOUT ME")
+        print(Fore.RESET + "\n 1. Personal information")
+        print(" 2. My Favorite Quotes")
+        print(" 3. Motivational Story")
+        print(" 4. Comment from Barcelos")
```
Comment (veslino-marc) [inline ourpackage/consultado.py:11]: I will add my comments. - Veslino

**Review by veslino-marc — CHANGES_REQUESTED** [review]

File: ourpackage/consultado.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
```
Comment (andaya-gener) [inline ourpackage/consultado.py:11]: I will add my comment. - Andaya

**Review by andaya-gener — CHANGES_REQUESTED** [review]

File: ourpackage/consultado.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        print(" 5. Comment from Veslino")
+        print(" 6. Comment from Andaya")
+        print(Fore.RED + "\n                    0 - Exit")
```
Comment (piadozo-edriane) [inline ourpackage/consultado.py:15]: I would like to add a comment about your module - Piadozo

**Review by piadozo-edriane — CHANGES_REQUESTED** [review]

File: ourpackage/consultado.py (hunk 1 above)
Comment (consultado-kirby) [reply ourpackage/consultado.py:15]: You have done it.

File: ourpackage/consultado.py (hunk 2 above)
Comment (consultado-kirby) [reply ourpackage/consultado.py:11]: You have done it.

File: ourpackage/consultado.py (hunk 3 above)
Comment (consultado-kirby) [reply ourpackage/consultado.py:11]: You have done it.

File: ourpackage/consultado.py (hunk 4 above)
Comment (consultado-kirby) [reply ourpackage/consultado.py:15]: You have done it.

**Review by andaya-gener — APPROVED** [review]

**Review by veslino-marc — APPROVED** [review]

**Review by barcelos-kevin — APPROVED** [review]

**Review by piadozo-edriane — APPROVED** [review]
Thank you :)

**Pull request merged**